media/css/firefox/whatsnew/whatsnew-66.scss

```diff
@@ -0,0 +1,457 @@
+
+        .mzp-c-button {
+            width: 100%;
+        }
+    }
+
+    @media #{$mq-sm} {
+        .fxa-email-field-container {
+            @include grid-column-gap($spacing-md);
+            display: grid;
+            grid-template-columns: 2fr 1fr;
+
+            .field {
+                margin-bottom: 0;
+            }
+        }
+    }
+
+    @media #{$mq-md} {
+        .fxa-email-form {
+            .agreement {
+                @include text-body-sm;
+            }
+        }
+
+        .fxa-email-field-container {
+            background: url('/media/img/firefox/whatsnew_63/avatar.svg') no-repeat;
+            @include bidi((
+                (background-position, center left, center right),
+                (padding-left, 37px + $spacing-md, padding-right, 0),
+            ));
+        }
+    }
+}
+
+//* -------------------------------------------------------------------------- */
+// Firefox newsletter form sticky header
+
+.c-sticky-signup.newsletter-form-cta {
+
+    // only display FxA form to users who are signed out.
+    .state-fxa-supported-signed-in & {
+        display: block;
+    }
+
+    .mzp-c-newsletter {
+        @include bidi(((text-align, left, right),));
+        display: block;
+        padding: 0;
+        margin: 0 auto;
+    }
+
+    .mzp-c-newsletter-form {
+        padding-top: 0;
+    }
+
+    .mzp-c-newsletter-content {
+        @include text-body-sm;
+    }
+
+    input[type="email"] {
+        padding: 18px 24px;
+        border: none;
+        box-shadow: none;
+    }
+
+    label[for="id_email"] {
+        @include visually-hidden;
+    }
+
+    p {
+        margin-bottom: $spacing-sm;
+    }
+
+    .mzp-c-form-submit {
+        margin-bottom: 0;
+    }
+
+    .mzp-c-fieldnote {
+        @include bidi(((text-align, left, right),));
+        color: $color-white;
+    }
+
+    .mzp-c-newsletter-thanks {
+        padding-top: 0;
+        text-align: center;
+
+        h3 {
+            @include text-display-sm;
+        }
+    }
+
+    @media #{$mq-md} {
+        .mzp-c-newsletter {
+            margin: 0;
+        }
+
+        .mzp-c-newsletter-content {
+            @include bidi(((padding-right, 40%, padding-left, 0),));
+            position: relative;
+        }
+
+        .mzp-c-button[type="submit"] {
+            @include bidi(((right, 0, left, auto),));
+            position: absolute;
+            top: 0;
+            width: calc(40% - #{$spacing-md});
+        }
+
+        .mzp-c-newsletter-thanks {
+            @include bidi(((text-align, left, right),));
+        }
+    }
+}
+
+// Dismiss
+
+.sticky-dismiss {
+    background: transparent;
+    border: 0;
+    bottom: 0;
+    cursor: pointer;
+    overflow: hidden;
+    padding: 0 0 0 $spacing-2xl;
+    position: absolute;
+    right: 0;
+    top: 5px;
+    width: 0;
+    height: 40px;
+
+    &:after {
+        @include text-body-lg;
+        background: transparent url('#{$image-path}/icons/ui/close-black.svg') center center no-repeat;
```

Comment:
The black close icon here has contrast issues against the blue background, would using the white variation be easier to see?